src/stderred.c

```diff
@@ -164,6 +194,14 @@ int FUNC(fprintf)(FILE *stream, const char *format, ...) {
   return result;
 }
 
+int FUNC(__fprintf_chk)(FILE *fp, int flag, const char *format, ...) {
```

Comment:
This function needs to be put behind a #ifdef check for existence since it is non standard.

See https://github.com/sickill/stderred/blob/master/src/CMakeLists.txt#L38-L50 for how